### PR TITLE
[Docs] Economy Cog Guide

### PR DESCRIPTION
**Pull request by bobloy**
### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### Description of the changes

Adds programmatically generated docs using the same method as #4488 etc.
Another step towards the #1734 goal

### CODE REVIEW

**Comment by bobloy** [comment]
Now ready for review.

File: docs/cog_guides/economy.rst [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+
+Examples:
+    - ``[p]bank set @Twentysix 26`` - Sets balance to 26
+    - ``[p]bank set @Twentysix +2`` - Increases balance by 2
+    - ``[p]bank set @Twentysix -6`` - Decreases balance by 6
+
+**Arguments**
+
+- ``<to>`` The user to set the currency of.
+- ``<creds>`` The amount of currency to set their alance to.
```
Comment (Flame442) [inline docs/cog_guides/economy.rst]:
```suggestion
- ``<creds>`` The amount of currency to set their balance to.
```

File: docs/cog_guides/economy.rst [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank set <to> <creds>
+
+**Description**
+
+Set the balance of user's bank account.
```
Comment (Flame442) [inline docs/cog_guides/economy.rst]:
```suggestion
Set the balance of a user's bank account.
```

File: docs/cog_guides/economy.rst [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+
+Passing positive and negative values will add/remove currency instead.
```
Comment (Flame442) [inline docs/cog_guides/economy.rst]: This is slightly unclear, since both `5` and `+5` are *technically* positive. The examples do make it clear however.

**Review by Flame442 — CHANGES_REQUESTED** [review]
Few fixes, but otherwise looking good!

*As always, the changes to the rst need to be applied to the relevant parts of the code.*

File: docs/cog_guides/economy.rst (hunk 3 above)
Comment (bobloy) [reply docs/cog_guides/economy.rst]: Added a clearer explanation

**Review by Flame442 — DISMISSED** [review]
All set! Thanks!

**Review by Flame442 — APPROVED** [review]

**Pull request merged**